app/models/review_assignment.rb

```diff
@@ -189,9 +191,10 @@ def contributor_to_review(reviewer, topic)
     raise "You have already reviewed all submissions for this #{work}." if contributor_set.empty?
 
     # Reduce to the contributors with the least number of reviews ("responses") received
+    # Reviewers should be able to review even if he completed all the min reviewed submissions.
     min_contributor = contributor_set.min_by {|a| a.responses.count }
     min_reviews = min_contributor.responses.count
-    contributor_set.reject! {|contributor| contributor.responses.count > min_reviews }
+    contributor_set.reject! {|contributor| contributor.responses.count > min_reviews + review_topic_threshold}
```

Comment:
This does not solve the problem stated in the project description.
Consider this case:
If A submit the artifact late and all the other artifacts all received more than 5 reviews. The max review can be done on each artifact is 10 and threshold is 2. In this case, the min_contributor will be 0, and the available contributors are the ones who got equal or less than 2 reviews. If so, the only reviewable work is A's own, and A cannot review his/her own work. So A still got nothing to review.